### PR TITLE
docs(accordion): decompose Disabled demo section

### LINKED ISSUE #3619: docs(accordion): decompose Disabled
Decompose `Disabled` section to `Disabled` and `Default opened` (related to #3128)

### PR DESCRIPTION
**Pull request by artolshansky**
close #3619

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

File: demo/src/app/components/+accordion/demos/disabled/disabled.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { Component } from '@angular/core';
 })
 export class DemoAccordionDisabledComponent {
   status: any = {
```
Comment (valorkin) [inline demo/src/app/components/+accordion/demos/disabled/disabled.ts]: why would you need object for only property?
please simplify

File: demo/src/app/components/+accordion/demos/opened/opened.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-accordion-opened',
+  templateUrl: './opened.html'
+})
+export class DemoAccordionOpenedComponent {
+  status: any = {
```
Comment (valorkin) [inline demo/src/app/components/+accordion/demos/opened/opened.ts]: same

**Review by valorkin — CHANGES_REQUESTED** [review]

**Comment by olshansky** [comment]
@valorkin fixed :)

**Review by IlyaSurmay — DISMISSED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**